specification/cognitiveservices/OpenAI.Inference/examples/2024-05-01-preview/start_generate_image.json

```diff
@@ -0,0 +1,25 @@
+{
+  "operationId": "BeginAzureBatchImageGeneration",
+  "title": "Starts the generation of a batch of images from a text caption",
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "api-version": "2024-04-01-preview",
```

Comment:
already on your radar, but we'll want these to match the 2024-05-01-preview in the folder path